app/src/main/java/com/rockthevote/grommet/data/db/model/RockyRequest.java

```diff
@@ -1,62 +0,0 @@
-package com.rockthevote.grommet.data.db.model;
-
-import android.os.Parcelable;
-import android.provider.BaseColumns;
-
-
-public abstract class RockyRequest implements Parcelable, BaseColumns {
-
-    public enum Status {
-        IN_PROGRESS("in_progress"),
-        ABANDONED("abandoned"),
-        FORM_COMPLETE("form_complete"),
-        REGISTER_SUCCESS("register_success"),
-        REGISTER_SERVER_FAILURE("register_server_failure"),
-        REGISTER_CLIENT_FAILURE("register_client_failure");
```

Comment:
I can't find a use for these. I'm assuming they were database column statuses and not API statuses. If so I can re-add it if we think we still need it.